etc/kayobe/environments/ci-multinode/inventory/group_vars/seed/stackhpc-dnf-repos

```diff
@@ -0,0 +1 @@
+dnf_custom_repos: "{{ stackhpc_dnf_repos }}"
```

Comment:
```suggestion
---
dnf_custom_repos: "{{ stackhpc_dnf_repos }}"

```

Reply:
Or perhaps move etc/kayobe/environments/ci-multinode/inventory/group_vars/overcloud/stackhpc-dnf-repos to etc/kayobe/environments/ci-multinode/inventory/group_vars/all/stackhpc-dnf-repos
This would also include infra-vms but I don't think that's a bad thing

Reply:
We still do not want this on the seed-hypervisor

Reply:
I'm not sure that we want this on the seed-hypervisor, as this would be less representative of a customer deployment.

Reply:
For a typical deployment there is a chicken and egg situation here because pulp runs on the seed. I agree that multinodes should reflect the standard config.

Reply:
What are we going to do here then? Why was this config necessary?

Reply:
This config was necessary to ensure RL 9.3 packages to be installed on seed when performing `kayobe seed host configure`